starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,41 @@
+body{
+  margin: 0;
+  background-color: : black
+}
+h1{
+  color: white;
+  text-align: center;
+  padding-top: 90px
+}
+
+.firstSection{
+  background: url("/Users/alexr/Desktop/coding/ironhack/firstWeek/secondAssignment/lab-css-recipes-clone/starter-code/apple-pie-starter-code/images/apple-pie.jpg") center center;
```

Comment:
This is the path that exists only in your local environment. When you work inside the project, it's enough to reference files and folders that are in the main folder so something like this works:
```
...
background: url("../images/apple-pie.jpg") center center;
...
```